port/win/env_win.cc

```diff
@@ -112,7 +112,8 @@ SSIZE_T pwrite(HANDLE hFile, const char* src, size_t numBytes,
 
   unsigned long bytesWritten = 0;
 
-  if (FALSE == WriteFile(hFile, src, numBytes, &bytesWritten, &overlapped)) {
+  if (FALSE == WriteFile(hFile, src, static_cast<DWORD>(numBytes), &bytesWritten,
```

Comment:
Please, add this before the call:
`assert(numBytes <= std::numeric_limits<DWORD>::max());`